fsButler/utils.py

```diff
@@ -59,29 +61,38 @@
 _zeroMagErrField = afwTable.Field["F"]("flux.zeromag.err",
                                        "The flux error corresponding to zero magnitude.")
 _stellarField = afwTable.Field["Flag"]("stellar",
-                                       "If true, the object is known to be a star if false it's known not to be a star.")
+                                       "If true, the object is known to be a star if false it's known not to\
+                                       be a star.")
```

Comment:
(Here and throughout) embedding new-lines in strings like this will cause weird spacing issues. Observe the difference:

```python
In [1]: a = ("123456789"
   ...:      "234567809")
   ...: print(a)
   ...:
123456789234567809

In [2]: b = ("123456789\
   ...:       234567809")
   ...: print(b)
   ...:
123456789      234567809
```